mapswipe_workers/mapswipe_workers/project_types/arbitrary_geometry/task.py

```diff
@@ -18,6 +18,9 @@ def __init__(self, group: object, featureId: int, featureGeometry: dict):
         super().__init__(group, taskId=task_id)
         self.geojson = featureGeometry
 
+        self.projectId = None
```

Comment:
is this removing project and group ID from all types of project? If so, I'd need to check that the app will work without it on `built_area` for instance

Reply:
After testing on the client, this actually causes a problem, because it is converted to a literal `"projectId":None` string in the output. I can obviously parse the resulting string to remove it (and groupId), but it seems like a waste of bandwidth to carry things around to then throw them out. Maybe `del self.projectId` instead?

Reply:
it's only removing groupId and projectId for footprint project types since this is in the `arbitrary_geometry/task.py`.

`arbitrary geometry` is the more general term of footprint I use for the back end.